src/teensy_sketch/teensy_sketch.ino

```diff
@@ -96,5 +142,137 @@ void runMainLoop() {
 
 void runInitLoop() {
   // TODO: ESP handshaking and blocking to wait on commands
+  while (1);
 }
 
+// ------------------------------------------------------------
+//                   Function definitions
+// ------------------------------------------------------------
+
+// REGION DATA COLLECTION FUNCTIONS
+float getAltitude(){
+  if (!altInit || altitudeCount == 256) {
+    D1 = helper.MS5637Read(ADC_D1, OSR);  // get raw pressure value
+    D2 = helper.MS5637Read(ADC_D2, OSR);  // get raw temperature value
+    dT = D2 - Pcal[5]*pow(2,8);    // calculate temperature difference from reference
```

Comment:
That seems to be the pattern throughout the file, `-` and `+` have spaces, but `*` and `/` do not. Is that a precedence convention?